src/app/shared/submission-section-cc-licenses/submission-section-cc-licenses.component.html

```diff
@@ -0,0 +1,143 @@
+<div class="mb-4 ccLicense-select">
+  <ds-select
+    [disabled]="!submissionCcLicenses"
+    [label]="'submission.sections.ccLicense.type'">
+
+    <ng-container class="selection">
+      <span *ngIf="!submissionCcLicenses">
+        <i class='fas fa-circle-notch fa-spin'></i>
```

Comment:
Why don't you use the `ds-loading` component in order to conform to the way dspace displays the loading state